tests/phpstan-polyfill.php

```diff
@@ -4,7 +4,5 @@
 
 // PHPStan does not read global constants from the stubs yet, remove this when it does
 if (defined('OCI_NO_AUTO_COMMIT')) {
-    return;
+    define('OCI_NO_AUTO_COMMIT', 0);
 }
-
-define('OCI_NO_AUTO_COMMIT', 0);
```

Comment:
:thinking: what does this have to do with long lines?

Reply:
There was another type of warning. I moved it to a separate commit.